### PR TITLE
Downgrading KCL to overcome AbstractMethodError

### PR DESCRIPTION
**Pull request by fredex42**
Running CAPI firehose library with KCL version 1.8.5 results in the following error when dealing with a closed shard:

```
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]: Caused by: java.lang.AbstractMethodError: Method com/gu/contentapi/firehose/ContentApiEventProcessor.shutdown(Lcom/amazonaws/services/kinesis/clientlibrary/interfaces/IRecordProcessorCheckpointer;Lcom/amazonaws/services/kinesis/clientlibrary/lib/worker/ShutdownReason;)V is abstract
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at com.gu.contentapi.firehose.ContentApiEventProcessor.shutdown(ContentApiFirehoseConsumer.scala)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at com.amazonaws.services.kinesis.clientlibrary.lib.worker.V1ToV2RecordProcessorAdapter.shutdown(V1ToV2RecordProcessorAdapter.java:48)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at com.amazonaws.services.kinesis.clientlibrary.lib.worker.ShutdownTask.call(ShutdownTask.java:103)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at com.amazonaws.services.kinesis.clientlibrary.lib.worker.MetricsCollectingTaskDecorator.call(MetricsCollectingTaskDecorator.java:49)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at com.amazonaws.services.kinesis.clientlibrary.lib.worker.MetricsCollectingTaskDecorator.call(MetricsCollectingTaskDecorator.java:24)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
Dec 04 11:09:15 ip-10-225-14-248 multimedia-launchdetector-v3[1362]:         ... 1 more
```

This then appears to break the main run loop with the following error, nixing the app.

```
SEVERE: Worker.run caught exception, sleeping for 2000 milli seconds!
```

This PR downgrades to a known good version of the KCL.

The reason appears to be that the definition of ShutdownReason has been moved from  `com/amazonaws/services/kinesis/clientlibrary/types/ShutdownReason` in 1.6 to  `com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShutdownReason` in 1.8

### CODE REVIEW

**Comment by fredex42** [comment]
confirmed that this fixed the problem on PROD.

**Comment by fredex42** [comment]
@tomrf1 - for some reason I can't directly request a review from you - would you mind approving? ta
Also, would you like me to make a PR to the crier client library project to make it compatible with the later KCL version?
@LATaylor-guardian - FYI this was the fix in the end.

**Comment by tomrf1** [comment]
👍 
that would be helpful, thanks!

File: multimedia-launchdetector-v3.ipr [hunk 1]
```diff
@@ -165,22 +165,22 @@
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
   <component name="libraryTable">
-    <library name="SBT: com.amazonaws:amazon-kinesis-client:1.8.5:jar">
+    <library name="SBT: com.amazonaws:amazon-kinesis-client:1.6.4:jar">
```
Comment (LATaylor-guardian) [inline multimedia-launchdetector-v3.ipr:168]: Should this file be checked in? what is it?

**Pull request merged**